routes/auth.js

```diff
@@ -7,10 +7,22 @@ const jwt = require('jsonwebtoken');
 const jwtDecode = require('jwt-decode');
 
 
-router.get('/users', passport.authenticate('jwt', { session: false }), async (req, res) => {
```

Comment:
I think we should have authentication enabled for all the implemented routes

Reply:
i can add it back